src/notifications/preferences.controller.ts

```
@@ -0,0 +1,29 @@
+import { CreatePreferencesDto } from './dto/create-preferences.dto';
+import { UpdatePreferencesDto } from './dto/update-preferences.dto';
+
+@Controller('notifications/preferences')
+export class PreferencesController {
+  constructor(private readonly preferencesService: PreferencesService) {}
+
+  @Post()
+  async createPreferences(
+    @Request() req,
+    @Body() dto: CreatePreferencesDto,
+  ) {
```

ESLint

[error] Replace `⏎····@Request()·req,⏎····@Body()·dto:·CreatePreferencesDto,⏎··` with `@Request()·req,·@Body()·dto:·CreatePreferencesDto`

(prettier/prettier)